superset/migrations/versions/f80a3b88324b_.py

```diff
@@ -0,0 +1,38 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Looks like this can be removed? I assume it was a placeholder for the migration I just ran

Reply:
I create this migration yesterday, but today you merge a new one. so both of us are based from same root. this file is to resolve the sequence problem for alembic. otherwise i have to re-create the migration script.

Reply:
I _think_ you could copy paste your migration into a new migration script and then not need this blank file. Sorry for the migration conflict!

Reply:
use `superset db merge heads` will generate migration script and fix the conflicts. no need to modify original file. There are many empty migration script in the codebase which was created for this purpose.